Topic8/src/main/java/com/aguirre/topic8/DemoApplication.java

```diff
@@ -0,0 +1,29 @@
+package com.aguirre.topic8;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.aguirre.topic8.controllers.StudentController;
+import com.aguirre.topic8.models.Student;
+import com.google.gson.Gson;
+
+@SpringBootApplication
+public class DemoApplication implements CommandLineRunner {
+
+	public static void main(String[] args) {
+
+		SpringApplication.run(DemoApplication.class, args);
+	}
+
+	@Override
+	public void run(String... arg0) throws Exception {
+		
+		ConfigurableApplicationContext context = new ClassPathXmlApplicationContext("application-context.xml");
+		
```

Comment:
Yes, I only have tested if the application context was loaded because that was my problem, after that I did some inserts and delete operations, but I deleted the lines, sorry, it remained incomplete.